code/modules/surgery/organs/organ_internal.dm

```diff
@@ -15,10 +15,13 @@
 		insert(holder)
 	..()
 
+
 /obj/item/organ/internal/proc/insert(mob/living/carbon/M, special = 0)
 	if(!iscarbon(M) || owner == M)
 		return
 
+	processing_objects -= src
```

Comment:
Isn't this going to cause problems for things that need to be processed every cycle, but aren't handled in an organ's `on_life` call?

For a known example (I'm sure there's others) body eggs have an independent process from `on_life` to tick their stage upward--isn't this going to get shortcircuited now when you insert them into people because it gets removed here?